### ISSUE TITLE
Inline code not the right font in preview mode

### ISSUE THREAD

**Issue by s-kyy** [sainadh-d/obsidian-boom#1]
### Steps to reproduce
1. Type some inline code: '` like so` '
2. Go into preview mode: CTRL + E

### Expected result
Font of inline code is as defined in the .css file. Hack I think?

### Actual result
Font of inline code is in a serif font (not the same as the default font everywhere else). 

### Environment
- Operating system: Windows 10
- Obsidian version: v0.6.2

----------

### Additional information

The obsidian.css file downloaded from the theme's GitHub was modified to include Focus Mode (search in the forum). Commenting it out does not affect the results in preview mode. 

Removing CSS plugin resolves this issue. 

Screenshot:
![Annotation 2020-05-25 164727](https://user-images.githubusercontent.com/19920083/82842428-7782a880-9ea7-11ea-9180-011fd1f72650.png)

**Comment by sainadhx** [comment]
@s-kyy Fixed this. Please check and let me know ?

**Comment by s-kyy** [comment]
I redownloaded your css and pasted it in the vault without any modifications.

It looks the same to me on my side. I tried it on a new note as well and it has the same behaviour too.

Preview mode:
![image](https://user-images.githubusercontent.com/19920083/83072907-8a2ee600-a03d-11ea-8bf6-09aa113a2950.png)

Editor mode:
![image](https://user-images.githubusercontent.com/19920083/83072927-93b84e00-a03d-11ea-98b7-e1888ca448ef.png)
